.vitepress/config.mts

```diff
@@ -35,6 +35,339 @@ function nav(): DefaultTheme.NavItem[] {
       link: '/reference/torii-endpoints',
       activeMatch: '/reference/',
     },
+    {
+      text: 'Cookbook',
+      link: '/cookbook/',
+      activeMatch: '/cookbook/',
+    },
+  ]
+}
+
+function sidebarCookbook(): DefaultTheme.SidebarItem[] {
+  return [
+    {
+      text: 'Access Control',
+      collapsed: true,
+      items: [
+        {
+          text: 'Permission Tokens',
+          collapsed: true,
+          items: [
+            {
+              text: 'Grant Permissions'
+            },
+            {
+              text: 'Revoke Permissions'
+            },
+            {
+              text: 'Mint Permission Tokens'
+            },
+            {
+              text: 'Burn Permission Tokens'
+            },
+          ],
+        },
+        {
+          text: 'Roles',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a Role',
+            },
+            {
+              text: 'Grant a Role',
+            },
+            {
+              text: 'Revoke a Role',
+            },
+          ],
+        },
+        {
+          text: 'Executors (Permission Validators)',
+          collapsed: true,
+          items: [
+            {
+              text: 'Write Executor',
+            },
+            {
+              text: 'Update Executor',
+            }
+          ],
+        }
+      ],
+    },
+    {
+      text: 'Accounts',
+      collapsed: true,
+      items: [
+        {
+          text: 'Register Accounts',
+        },
+        {
+          text: 'Unregister Accounts',
+        },
+        {
+          text: 'Update Account Metadata'
+        },
+        // ...?
+      ],
+    },
+    {
+      text: 'Assets',
```

Comment:
we don't have a section on asset definition?

Reply:
no
should we add it within the Assets section? or a completely separate one?

Reply:
I would make a separate one. With Regsiter/Unregister/Transfer